apps/src/p5lab/AnimationPicker/AnimationPicker.jsx

```diff
@@ -77,6 +79,10 @@ class AnimationPicker extends React.Component {
         libraryManifest={this.props.libraryManifest}
         hideUploadOption={this.props.hideUploadOption}
         hideAnimationNames={this.props.hideAnimationNames}
+        navigatable={!this.props.isBackground}
+        defaultCategoryQuery={this.props.isBackground ? 'backgrounds' : null}
+        hideBackgrounds={!this.props.isBackground && this.props.isSpriteLab}
```

Comment:
i think we should move this up to `<P5LabView/>` as well, and make it a generic "filter" function for categories so that `<AnimationPickerBody/>` is more easily configurable in this way.

```javascript
// within P5LabView
filterBackgrounds = categories => {
  // filter out backgrounds
}

<AnimationPicker
  categoryFilter={this.filterBackgrounds}
  ...
/>
```